openapi3/swagger_loader_relative_refs_test.go

```diff
@@ -113,6 +113,17 @@ var refTestDataEntries = []refTestDataEntry{
 			require.Equal(t, "id", swagger.Paths["/test/{id}"].Get.Parameters[0].Value.Name)
 		},
 	},
+
+	{
+		name:            "PathOperationParameterRefWithContentInQuery",
+		contentTemplate: externalPathOperationParameterWithContentInQueryTemplate,
+		testFunc: func(t *testing.T, swagger *openapi3.Swagger) {
+			fmt.Print(swagger.Paths["/test/{id}"].Get.Parameters[0].Value.Content["application/json"].Schema.Ref)
+			require.NotNil(t, swagger.Paths["/test/{id}"].Get.Parameters[0].Value.Content["application/json"].Schema.Value)
```

Comment:
Could you put this value in a variable & reuse it below?
Also remove the debug line above.

Reply:
ok ,I will delete debug code . but i don't understand what value i can put in a variable and reuse it? i think i totally follow  style of your code ! could you give me a detailed remind ?

Reply:
Oh it’s just for DRY. Note that i didn’t write most of this code so if the style is to repeat many things then that is not something I agree with...

Reply:
well ! please review my new commit .